FORMAT.md

```diff
@@ -118,29 +118,43 @@ The bracket element refers to a rigid body that connects modules, such as a ligh
   - R8HeavyRightInside
   - R8HeavyRightOutside  
 
+Currently, all of the given bracket elements have a single output interface.
```

Comment:
If we define brackets that have two outputs in the future (not double module brackets, but full "Y"/etc brackets), we can update this.

The initial areas I would expect to make sense here are adding "chassis" types as brackets, so you have the hexapod chassis or the rosie chassis, for example.